exspy/components/eels_cl_edge.py

```diff
@@ -146,7 +147,7 @@ class EELSCLEdge(Component):
     _fine_structure_coeff_free = True
     _fine_structure_spline_active = True
 
-    def __init__(self, element_subshell, GOS="gosh", gos_file_path=None):
+    def __init__(self, element_subshell, GOS="dft", gos_file_path=None):
```

Comment:
A side note as this is outside of the scope of this PR: I find it confusing to have both `GOS` and `gos_file_path`. I am wondering if these should be merged together to make `GOS` accept one of the supported string (`"dft", "dirac", etc.) or a path (which could a str or `pathlib.Path`) in which case, it will try to read the file assuming this is gosh file?

Reply:
Agree, this makes sense to me.